### PR TITLE
[#102] Remove old project and add empty nimble templates

### LINKED ISSUE #102: As an iOS developer, I can create a new iOS project from custom project template
- [x] Create an **empty** custom project template
- [x] Create a script to copy the template to local template folder (~/Library/Developer/Xcode/Templates/)

### PR DESCRIPTION
**Pull request by suho**
https://github.com/nimbl3/ios-templates/issues/102

Resolve #102 

## What happened

Create install script and empty templates based on Apple's templates
 
## Insight

- Create an empty custom project template
- Create a script to copy the template to local template folder (~/Library/Developer/Xcode/Templates/)

## Proof Of Work

Screen Shot 2020-09-24 at 09 44 28

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p ~/Library/Developer/Xcode/Templates/File\ Templates
+mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates
+mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates/Nimble\ Templates
```
Comment (blyscuit) [inline install.sh:5]: what is the following for? If we need more files, we can make sub folders inside our template.
```
mkdir -p ~/Library/Developer/Xcode/Templates/File\ Templates
 mkdir -p ~/Library/Developer/Xcode/Templates/Project\ Templates
```

**Comment by blyscuit** [comment]
Screen Shot 2020-09-28 at 19 16 15

**Comment by suho** [comment]
@blyscuit Please separate your commit into another branch, and create a pull request with base branch is `feature/102-empty-templates` for easy to track and follow

**Comment by blyscuit** [comment]
Ok, I misunderstand comments in issue. 👍

File: install.sh (hunk 1 above)
Comment (suho) [reply install.sh:5]: I just followed from Apple's Templates, in Templates folder from Xcode, it has `File Templates` and `Project Templates` folders. So if we need more files, I think we will add file template into `File Templates/Nimble Templates`. What do you think?

Comment (blyscuit) [reply install.sh:5]: We could create `~/Library/Developer/Xcode/Templates/Nimble\ Templates` and have subfolders inside that?

Comment (suho) [reply install.sh:5]: If we create a subfolder (e.g `File Templates` or `Project Templates`), the name of section when create new project or file is not showing in the right name.

Screenshot_29_9_20_12_12

ContentView_swift

So I suggest: 
```
~/Library/Developer/Xcode/Templates/File Templates/Nimble Templates
~/Library/Developer/Xcode/Templates/Project Templates/Nimble Templates
```

So we can separate file and project templates into 2 folders same like Apple did

Screen Shot 2020-09-29 at 12 14 35

What do you think?

**Review by blyscuit — APPROVED** [review]

**Review by vnntsu — APPROVED** [review]

**Pull request merged**